src/Phinx/Console/Command/Create.php

```diff
@@ -76,6 +89,10 @@ protected function getCreateMigrationDirectoryQuestion()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
+        /**
+         * @var CreationInterface $creationClass
```

Comment:
Which is pretty rubbish really. You should be able to docblock vars in any scope. PHPStorm is happy with this, but I'll remove it